### PR TITLE
Add "Interactions with players" golden rule and clarification (draft)

### PR DESCRIPTION
**Pull request by Nepiki**
Add a golden rule describing that developers should keep the same amount of respect for users that they expect, with additional clarification. Included is a step for developers in case they are dealing with harassment.

### CODE REVIEW

File: docs/guidelines/developers/code-of-conduct.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ This document covers developers' rights, responsibilities, and rules for fair pl
 - Respect your fellow developer and communicate through any issues you have civilly.
 - Make room for other developers to get a chance to work on games they like, in their own way.
 - Do not discourage anyone from working on any game, in public or in private.
+- As players are expected to treat developers with respect, likewise it's the responsibility of a developer to ensure players receive the same amount of respect in return.
+
+## Interactions with players
+
+Interactions between players and developers may not always go as smoothly as desired. The User Code of Conduct exists to ensure players keep a healthy attitude towards developers, and likewise developers are required to do the same. Don't diminish a user's respectful feedback or interaction as harmful, and instead try to understand them and see what can be learned from it. The viewpoint of a player can be different than that of a developer with experience of how development works, so it is important they understand why certain choices are made they may disagree with.
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]: Recommendations for conciseness and clarity.
```suggestion
Interactions between players and developers may not always go smoothly. The User Code of Conduct exists to ensure players keep a healthy attitude towards developers, and likewise developers are required to do the same. Don't dismiss a user's respectful feedback as harmful or invalid. Instead, try to understand them and see what can be learned from it. A player's viewpoint can be very different than a developer's, so it is important they understand why certain choices are made they may disagree with.
```
May also want to link directly to the user CoC.

File: docs/guidelines/developers/code-of-conduct.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ This document covers developers' rights, responsibilities, and rules for fair pl
 - Respect your fellow developer and communicate through any issues you have civilly.
 - Make room for other developers to get a chance to work on games they like, in their own way.
 - Do not discourage anyone from working on any game, in public or in private.
+- As players are expected to treat developers with respect, likewise it's the responsibility of a developer to ensure players receive the same amount of respect in return.
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]:
```suggestion
- Developers are expected to treat players with respect, just as players are expected to treat developers with respect.
```

File: docs/guidelines/developers/code-of-conduct.md [hunk 3]
```diff
@@ -26,6 +26,13 @@ This document covers developers' rights, responsibilities, and rules for fair pl
...
+
+## Interactions with players
+
+Interactions between players and developers may not always go as smoothly as desired. The User Code of Conduct exists to ensure players keep a healthy attitude towards developers, and likewise developers are required to do the same. Don't diminish a user's respectful feedback or interaction as harmful, and instead try to understand them and see what can be learned from it. The viewpoint of a player can be different than that of a developer with experience of how development works, so it is important they understand why certain choices are made they may disagree with.
+
+If the user goes on the offensive and starts harassing a developer or their sets, please don't engage further with them and instead reach out to RAdmin or anyone part of the mod team whenever you can. Harassment is taken seriously, and actions against the user will be taken when deemed necessary. The moderation team exists to ensure you can keep developing without your enjoyment being harmed by unwarranted negative interactions.
```
Comment (wescopeland) [inline docs/guidelines/developers/code-of-conduct.md]: Recommendations for clarity and accuracy.
```suggestion
If a user goes on the offensive and starts harassing you or your sets, please don't engage further with that user. Instead, reach out to the moderation team or RAdmin. Harassment is taken seriously, and actions against the user, such as a mute or even a site ban, will be taken when deemed necessary. The moderation team will ensure you can keep developing without your enjoyment being harmed by unwarranted negative interactions.
```

**Review by wescopeland — COMMENTED** [review]
Hi @Nepiki,

LGTM. Mostly recommendations here for clarity.

**Comment by Nepiki** [comment]
All good recommended changes, they should all be corrected now. Also agreed on User's Code of Conduct link, which has not been added yet

**Comment by Nepiki** [comment]
Truly all are corrected now (my bad, still learning ☹️ ), and User COC link has been added.

**Review by wescopeland — APPROVED** [review]
Not a problem at all, thank you!

**Pull request merged**